HW5/SikorskaViktoriia/lab5.1.py

```diff
@@ -0,0 +1,9 @@
+list_int = list((input("Enter integers number separated by spaces  -> ")))
```

Comment:
Incorrect input handling

input("Enter integers number separated by spaces -> ") returns a string, and list(...) converts it into a list of characters instead of numbers.

For example, if the user enters 12 34 56, list_int becomes ['1', '2', ' ', '3', '4', ' ', '5', '6'], which is incorrect.

Unnecessary filtering with if item.strip()

This removes spaces, but the real issue is that spaces should not be included in the list in the first place.

Incorrect float conversion

The code modifies clean_list_int in a loop, but it's better to create a new list directly.





Fixed version of the code:

list_int = input("Enter integers separated by spaces -> ").split()

clean_list_int = [float(item) for item in list_int]

print(clean_list_int, type(clean_list_int))